### PR TITLE
Create ubuntu_18_04.adoc

### PR DESCRIPTION
**Pull request by voroyam**

### CODE REVIEW

**Comment by mmattel** [comment]
@voroyam 
we already have a good ubuntu 18.04 guide, why writing another one?
https://doc.owncloud.org/server/admin_manual/installation/server_prep_ubuntu_18.04.html

**Comment by voroyam** [comment]
@mmattel The goal was to create a copy pastable simple guide to install ownCloud for non expert users

**Comment by settermjd** [comment]
I might well have found the reason behind why the build's failing. It's failing when running `yarn validate`. I thought, at first, that there was an issue with yarn. However, running `yarn validate` or running antora directly, both return an exit code of 1. However, when running `yarn antora`, the following error is displayed:

```
Error: Page alias cannot reference itself: master@server:admin_manual:installation/command_line_installation.adoc
    at ContentCatalog.registerPageAlias (/Users/settermjd/Workspace/clients/ownCloud/docs/node_modules/@antora/content-classifier/lib/content-catalog.js:176:15)
    at aliases.split.forEach (/Users/settermjd/Workspace/clients/ownCloud/docs/node_modules/@antora/document-converter/lib/convert-document.js:43:57)
    at Array.forEach (<anonymous>)
    at registerAliases (/Users/settermjd/Workspace/clients/ownCloud/docs/node_modules/@antora/document-converter/lib/convert-document.js:43:6)
    at convertDocument (/Users/settermjd/Workspace/clients/ownCloud/docs/node_modules/@antora/document-converter/lib/convert-document.js:30:3)
    at contentCatalog.findBy.filter.map (/Users/settermjd/Workspace/clients/ownCloud/docs/node_modules/@antora/document-converter/lib/convert-documents.js:24:58)
    at Array.map (<anonymous>)
    at convertDocuments (/Users/settermjd/Workspace/clients/ownCloud/docs/node_modules/@antora/document-converter/lib/convert-documents.js:24:6)
    at generateSite (/Users/settermjd/Workspace/clients/ownCloud/docs/generator/generate-site.js:27:17)
    at <anonymous>
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

File: modules/admin_manual/pages/installation/linux_installation.adoc [hunk 1]
```diff
@@ -1,5 +1,6 @@
 = Linux Package Manager Installation
 :toc: right
+:page-aliases: installation/linux_installation.adoc
```
Comment (settermjd) [inline modules/admin_manual/pages/installation/linux_installation.adoc]: This creates a circular reference to itself.

File: modules/admin_manual/pages/installation/installation_wizard.adoc [hunk 2]
```diff
@@ -1,5 +1,6 @@
 = The Installation Wizard
 :toc: right
+:page-aliases: installation/installation_wizard.adoc
```
Comment (settermjd) [inline modules/admin_manual/pages/installation/installation_wizard.adoc]: This creates a circular reference to itself.

File: modules/admin_manual/pages/installation/command_line_installation.adoc [hunk 3]
```diff
@@ -1,4 +1,5 @@
 = Command Line Installation
+:page-aliases: installation/command_line_installation.adoc
```
Comment (settermjd) [inline modules/admin_manual/pages/installation/command_line_installation.adoc]: This creates a circular reference to itself.

**Comment by mmattel** [comment]
@voroyam there is no script like setup -  wet dream ...
but the instruction in the referenced guide is really a very good one as it covers all things structured in one document. this is already copy paste. but brainless does not work in setting up a environment. this is not windows...

**Comment by settermjd** [comment]
Secondly, this PR seems to be a duplicate of https://github.com/owncloud/docs/blob/master/modules/admin_manual/pages/installation/server_prep_ubuntu_18.04.adoc.

**Comment by voroyam** [comment]
> there is no script like setup - wet dream ...

name me one thing that does not work in this pr. @mmattel 

Also referencing a document that is not listed but expecting that everyone knows it's there, it's kinda weird.

**Comment by settermjd** [comment]
Here's where the document is linked to in the docs

link to ubuntu 18 04 preparation guide

**Comment by voroyam** [comment]
> Here's where the document is linked to in the docs
> 
> link to ubuntu 18 04 preparation guide

okay, you found the link but probably because you put it there :)

If I am in a meeting with the whole customer success team and no one ever saw the document you are referencing - it's has not been seen.

**Comment by settermjd** [comment]
@voroyam, I found it because it's in that section of the docs. Let me know your suggestions for improving its visibility. Then let's go from there.

**Comment by voroyam** [comment]
I don't know that document neither was I or another member of the support team involved in creating it.

The new document, this PR's one, is one which was created by the support team in a script format. This one we want to be in the docs, included in the Table of Content. 

I can understand that it's not optimal that a document which was created by you and @mmattel does not get noticed, but that's the danger in working outside the team.

**Comment by JKawohl** [comment]
I really like the idea of a copypasta setup within minutes and without "prosaic" explanations

**Comment by JKawohl** [comment]
maybe we could do this c&p for the 4 biggest distros: 
Ubuntu, Debian, CentOS, and openSUSE.

Most users are interested in a simple setup. As we don't ship Linux Packages with all dependencies (which i think is correct for a php webapplication, .....dependency hell and the possibility of breaking other services....yada yada yada,) we should make shure that installing owncloud is as simple as possible.

**Comment by voroyam** [comment]
> maybe we could do this c&p for the 4 biggest distros:
> Ubuntu, Debian, CentOS, and openSUSE.

As discussed at the Meeting - we want our official docs to be Ubuntu exclusive. The other how-tos will be added to kb. I already talked to Erik regarding the CentOS installation

**Pull request merged**